tests/unit_tests/include/test_helper.hpp

```diff
@@ -238,16 +232,15 @@
             if (vendor_id == INTEL_ID)                                     \
                 TEST_RUN_INTELGPU_SELECT(q, func, __VA_ARGS__);            \
             else if (vendor_id == NVIDIA_ID) {                             \
-                TEST_RUN_NVIDIAGPU_CUBLAS_SELECT(q, func, __VA_ARGS__);    \
+                TEST_RUN_NVIDIAGPU_CURAND_SELECT(q, func, __VA_ARGS__);    \
             }                                                              \
             else if (vendor_id == AMD_ID) {                                \
-                TEST_RUN_AMDGPU_ROCBLAS_SELECT(q, func, __VA_ARGS__);      \
+                TEST_RUN_AMDGPU_ROCRAND_SELECT(q, func, __VA_ARGS__);      \
```

Comment:
Do changes to this file belong in this PR? They don't appear to be Sparse BLAS related.

Reply:
Yes I noticed git makes the diff look more complicated than it should be. This is only removing an unused macro `TEST_RUN_CT_SELECT`. The macros from the other domains `TEST_RUN_BLAS_CT_SELECT`, `TEST_RUN_RNG_CT_SELECT`, and `TEST_RUN_LAPACK_CT_SELECT` are unchanged. I think it's fine to remove dead code as part of this PR but I can make it separate if you prefer.